### ISSUE TITLE
This panel does not go down when I have to choice a favorite location

### ISSUE THREAD

**Issue by serrq** [bmaroti9/Overmorrow#31]
Android 14
OnePlus CPH2399
Overmorrow 2.4.1 (Izzy)

![1000118282](https://github.com/user-attachments/assets/5939a53a-81e4-4651-9827-a269c7687192)

**Comment by bmaroti9** [comment]
This is kind of intentionally like this... The whole point is that the search screen appears where the image was. And anyway it doesn't make much difference because even if it were expanded, the keyboard would cover it.

**Comment by bmaroti9** [comment]
Try to set the search provider to open-meteo (settings/general/search provider).

**Comment by bmaroti9** [comment]
True. I am planning to do something about that but I'm still not sure what. I'm thinking of pages that you could swipe through, but a scrollable column could work as well. I haven't decided yet.

**Comment by bmaroti9** [comment]
Hi!
So I know it's been a long time, but I've finally found a solution to expand it all the way. The solution was to make the original search bar act more like a button, that when pressed will bring you to the full screen search page, but i added some animations and stuff to make it look as though it's still the same bar.
Having all this new space, opened up lot's of new possibilities, so I did a total redesign of the search page (partially inspired by google's new weather app)

https://github.com/user-attachments/assets/5e687ac9-8ae9-4124-9611-5ffcc01a1405

This in turn inspired me to do a total overhaul of the ui (again), so you can see some changes there too.
It should be available in the next release (though i think that's still far away because i'm still in the middle of the ui overhaul)
